tests/migrations/test_runner_individual.py

```diff
@@ -0,0 +1,236 @@
+from snuba.clickhouse.http import JSONRowEncoder
```

Comment:
Are there any changes here at all? Or is it just moving the file to a new location?

Reply:
There are some more changes (new functions) now!